### PR TITLE
Integrate api

### PR DESCRIPTION
**Pull request by gitahievans**
In this pull request I have created a component that fetches data from the OpenWeatherMap API and tests whether the API works.

### CODE REVIEW

File: src/components/GetApi/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from "react";
+import axios from "axios";
+
+const Index: React.FC = () => {
+  useEffect(() => {
+    weatherData();
+  }, []);
```
Comment (daveclinton) [inline src/components/GetApi/index.tsx]: specify dependencies so that the effect runs on every render

File: src/components/GetApi/index.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      let response = await axios.get(
+        `https://api.openweathermap.org/data/2.5/weather?q=New York&appid=5a6a3604d50f420072819316b2ed23a2`
+      )
+      console.log(response)
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return <div></div>;
```
Comment (daveclinton) [inline src/components/GetApi/index.tsx]: WHat's the use of the jsx component here?

**Review by daveclinton — CHANGES_REQUESTED** [review]

File: src/components/GetApi/index.tsx (hunk 2 above)
Comment (gitahievans) [reply src/components/GetApi/index.tsx]: I have gotten rid of that one

File: src/components/GetApi/index.tsx (hunk 1 above)
Comment (gitahievans) [reply src/components/GetApi/index.tsx]: Okay

**Pull request merged**